### PR TITLE
#166879999 Users should be able to rate articles

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
#### What does this PR do?
Users should be able to rate articles

#### Description of Task to be completed?
A 5-star rating system is the end goal. Let's have authenticated users get visual feedback of the average rating of their articles

#### How should this be manually tested?
Follow the setup steps on the [readme page](https://github.com/andela/ah-frontend-kronos) of this repo to set-up locally, you will notice star ratings on the articles and are able to rate when reading the article.

#### What are the relevant pivotal tracker stories?
[#166879999](https://www.pivotaltracker.com/story/show/166879999)

#### Screenshots (if appropriate)
Screenshot 2019-07-26 at 10 38 45

### CODE REVIEW

File: src/actions/articles/rating/rateArticleAction.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable func-names */
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { toastSuccess, toastFailure } from '../../../utils/toast';
+import { ArticleConstants } from '../../actionTypes';
+
+const apiUrl = 'https://ah-backend-kronos-staging.herokuapp.com/api/articles/';
+const lastPart = '/rate';
```
Comment (reifred) [inline src/actions/articles/rating/rateArticleAction.js:8]: I think we have url connections file that stores the url there is no need to declare the url again.

File: src/actions/articles/rating/updateRatingAction.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable func-names */
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { toastSuccess, toastFailure } from '../../../utils/toast';
+import { ArticleConstants } from '../../actionTypes';
+
+const apiUrl = 'https://ah-backend-kronos-staging.herokuapp.com/api/articles/';
+const lastPart = '/rate';
```
Comment (reifred) [inline src/actions/articles/rating/updateRatingAction.js:8]: The same at this point don't re-declare the url

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks @Manorlds-Eaglespark for the work done. However, I got an error after rating an article maybe you should consider correcting this.
image

**Comment by Manorlds-Eaglespark** [comment]
> Thanks @Manorlds-Eaglespark for the work done. However, I got an error after rating an article maybe you should consider correcting this.
> image
Ohh, I  forget to clean up resources properly after unmount. Thanks.

**Comment by reifred** [comment]
> > Thanks @Manorlds-Eaglespark for the work done. However, I got an error after rating an article maybe you should consider correcting this.
> > image
> 
> You could try refreshing your browser.

I did kindly look into it and notify me when it is solved

**Comment by Manorlds-Eaglespark** [comment]
> > > Thanks @Manorlds-Eaglespark for the work done. However, I got an error after rating an article maybe you should consider correcting this.
> > > image
> > 
> > 
> > You could try refreshing your browser.
> 
> I did kindly look into it and notify me when it is solved

Sure, alright.

**Comment by Manorlds-Eaglespark** [comment]
> > > > Thanks @Manorlds-Eaglespark for the work done. However, I got an error after rating an article maybe you should consider correcting this.
> > > > image
> > > 
> > > 
> > > You could try refreshing your browser.
> > 
> > 
> > I did kindly look into it and notify me when it is solved
> 
> Sure, alright.

I have done something to address this. Though it is not in the scope of this feature. Thanks.

**Review by reifred — APPROVED** [review]
Thanks for the good work done @Manorlds-Eaglespark

**Review by Rhytah — APPROVED** [review]

**Review by kcharles52 — APPROVED** [review]

**Pull request merged**